src/onemod/constraints/base.py

```diff
@@ -0,0 +1,78 @@
+from typing import Any, Callable, Dict
+from polars import Series
+
+from pydantic import BaseModel, field_validator
+
+from .functions import bounds, is_in
```

Comment:
What's the difference between importing with `.functions` and `onemod.constraints.functions`?

Reply:
no difference, stylistic thing